stored_requests/multifetcher.go

```diff
@@ -35,6 +35,10 @@ func (mf MultiFetcher) FetchRequests(ctx context.Context, requestIDs []string, i
 	return
 }
 
+func (mf MultiFetcher) FetchCategories(primaryAdServer, publisherId, iabCategory string) (string, error) {
+	return "", nil
```

Comment:
All the possible fetchers already have nil returns if they don't support category fetching. The `MultiFetcher` should just loop over the list of fetchers and combine them as it does in `FetchRequests()`, so that the multifetcher will work as more category fetchers are added.

Reply:
Good catch, changed this function to loop through the list of fetchers.